docs/conf.py

```diff
@@ -99,25 +72,25 @@ def setup(app):
 # The master toctree document.
 master_doc = 'index'
 
-
 # This allows modules to be indexed under the submodule name rather than all appearing under sksurgeryvtk
 modindex_common_prefix = [
     'sksurgeryvtk.'
 ]
 
 # General information about the project.
 project = u'scikit-surgeryvtk'
-copyright = u"2018, University College London"
+copyright = u"2023, University College London"
+# Authors: Stephen Thompson, Matt Clarkson, Thomas Dowrick and Miguel Xochicale
 author = u'Thomas Dowrick'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u''
+version = u'2.0.0'
 # The full version, including alpha/beta/rc tags.
-release = u''
+release = u'Final release'
```

Comment:
Does it make sense to call it 'Final release'?. There will be future releases.

Reply:
You are right. just pushed commit 'setting same value for version and release in docs/conf.py > https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version (#216, #217)'

Let me know if you have other comments or feedback. 

Thanks